src/dataconnect/fileUtils.ts

```diff
@@ -43,6 +40,9 @@ export async function readDataConnectYaml(sourceDirectory: string): Promise<Data
 
 function validateDataConnectYaml(unvalidated: any): DataConnectYaml {
   // TODO: Add validation
+  if (!unvalidated["location"]) {
+    throw new FirebaseError("Missing required field 'location' in dataconnect.yaml");
+  }
   return unvalidated as DataConnectYaml;
```

Comment:
OOC, Q: do we want to validate other fields as well?

Is there a way to make json schema do it for us?

Reply:
That's a good point, and we defiitely should leverage the json schema here. Adding a todo for now.